lib/fs.js

```diff
@@ -108,46 +107,17 @@ function copyObject(source) {
   return target;
 }
 
-// TODO(joyeecheung): explore how the deprecation could be solved via linting
```

Comment:
I am wondering if there is a linting rule for this...? Could not find one in https://standardjs.com/rules.html at least.

Reply:
There is no such rule in eslint at the moment. We could of course create one and add that to eslint. I am going to open an issue there about it.

Reply:
https://github.com/eslint/eslint/issues/9971

Reply:
@joyeecheung if you want to get something like that in eslint, I guess we have to get some likes under my suggestion ;-)